lib/Twig/TemplateWrapper.php

```diff
@@ -99,6 +99,8 @@ public function renderBlock($name, $context = array())
      *
      * @param string $name    The block name to render
      * @param array  $context An array of parameters to pass to the template
+     *
+     * @return string
```

Comment:
not sure it adds value, but if you add it here, it should be added to `display()` as well (in the same file).